pkg/summary/options.go

```diff
@@ -0,0 +1,51 @@
+package summary
+
+import (
+	"github.com/fairwindsops/goldilocks/pkg/kube"
+	"github.com/fairwindsops/goldilocks/pkg/utils"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+// Functional options
+type Option func(*Options)
+
+// internal options for getting and caching the Summarizer's VPAs
+type Options struct {
```

Comment:
If these are internal options only, then should the type be un-exported?

Reply:
I've changed this to not be exported.